### ISSUE TITLE
Does not show common, uncommon, rare and mythic amounts in "my deck" in version 3.0.1

### ISSUE THREAD

**Issue by ZXA999** [Manuel-777/MTG-Arena-Tool#841]
In version 2.14.4 you could see all the common, uncommon, rare and mythic required to complete the deck. In version 3.0.1 you can only see the Boosters needed to complete. Why was such an important resource withdrawn?

**Comment by Manuel-777** [comment]
It was not removed, so it's probably just a display issue on the decks view/table, is the toggle enabled for that row?

**Comment by ZXA999** [comment]
> 
> 
> It was not removed, so it's probably just a display issue on the decks view/table, is the toggle enabled for that row?

I tried to enable / disable everything I found in "My Decks", but found nothing.
I'm glad it wasn't removed, but I don't know how I can view it.

**Comment by lusbenjamin** [comment]
Sorry for the confusion! The "Booster Cost" column shows _BOTH_ the estimated required number of boosters and the breakdown of each count of wildcard required by rarity. It was named "Booster Cost" in an attempt to make it clear that filters and sorts would be based solely on the booster number. (It is definitely possible to add individual columns for each WC in a future build.)

**Comment by ZXA999** [comment]
I have several decks that need W.C to complete but I already have all the necessary boosters, when I use the "wanted" filter it's like I've already completed everything. Same with several W.C missing.

**Comment by lusbenjamin** [comment]
solved in #835

**Comment by ZXA999** [comment]
> 
> 
> solved in #835

I read # 835 and it really was a great job, congratulations !!!
I'm looking forward to testing it, is it available in version 3.03?

**Comment by lusbenjamin** [comment]
No, this is a major feature build that will most likely ship as 3.1.0 in the next month or so.

**Comment by ZXA999** [comment]
I understand, thank you very much for the clarifications.
